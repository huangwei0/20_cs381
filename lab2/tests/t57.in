debug off
stopwords on
load tracks.txt
preprocess
length 1
sequence A
length 2
sequence B
length 3
sequence C
length 4
sequence D
length 5
sequence E
length 6
sequence F
length 7
sequence G
length 8
sequence H