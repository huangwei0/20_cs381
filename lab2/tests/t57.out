a 
b flat major
c major op 11
d minor op 11 bagatelles
e flat major op 11 bagatelles
f major op 11 bagatelles solo tu
g minor op 11 bagatelles solo tu amor
h binds o come back home christmas song you
